boards/arm/cy8cproto_062_4343w/cy8cproto_062_4343w.dts

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2021 Cypress Semiconductor Corporation.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <infineon/psoc6/mpns/CY8C624ABZI_S2D44.dtsi>
+#include <infineon/psoc6/system_clocks.dtsi>
+#include "cy8cproto_062_4343w_common.dtsi"
+
+/ {
+	model = "cy8cproto_062_4343w with an Cypress PSoC™ 6 SoC";
+	compatible = "cy8cproto_062_4343w", "PSoC6";
+
+	aliases {
+		uart-5 = &uart5;
+		i2c-0  = &i2c3;
+	};
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,console = &uart5;
+		zephyr,shell-uart = &uart5;
+	};
+};
+
+&counter0_0 {
+	status = "okay";
+};
+
+&uart5 {
+	status = "okay";
+	current-speed = <115200>;
+
+	/* UART pins */
+	pinctrl-0 = <&p5_1_scb5_uart_tx &p5_0_scb5_uart_rx>;
+	pinctrl-names = "default";
+};
+
+&i2c3 {
+	status = "okay";
+
+	/* I2C pins */
+	pinctrl-0 = <&p6_0_scb3_i2c_scl &p6_1_scb3_i2c_sda>;
+	pinctrl-names = "default";
+};
+
+/* System clock configuration */
+&fll0 {
+	status = "okay";
+	clock-frequency = <100000000>;
+};
+
+&clk_hf0 {
+	clock-div = <1>;
+	clocks = <&fll0>;
+};
+
+/* CM4 core clock = 100MHz
+ * &fll clock-frequency / &clk_hf0 clock-div / &clk_fast clock-div = 100MHz / 1 / 1 = 100MHz
+ */
+&clk_fast {
+	clock-div = <1>;
+};
+
+/* CM0+ core clock = 50MHz
+ * &fll clock-frequency / &clk_hf0 clock-div / &clk_slow clock-div = 100MHz / 1 / 2 = 50MHz
+ */
+&clk_slow {
+	clock-div = <2>;
+};
+
+/* PERI core clock = 100MHz
+ * &fll clock-frequency / &clk_hf0 clock-div / &clk_peri clock-div = 100MHz / 1 / 1 = 100MHz
+ */
+&clk_peri {
+	clock-div = <1>;
+};
+
+&pinctrl {
+
+	p3_3_scb2_uart_cts {
+		input-enable;
+	};
+
+	/* Configure pin control bias mode for uart5 pins */
+	p5_1_scb5_uart_tx {
+		drive-push-pull;
+	};
+
+	p5_0_scb5_uart_rx {
+		input-enable;
+	};
+
+	/* Configure pin control bias mode for i2c3 pins */
+	p6_0_scb3_i2c_scl {
+		drive-open-drain;
+		input-enable;
+	};
+
+	p6_1_scb3_i2c_sda {
+		drive-open-drain;
+		input-enable;
+	};
+};
```

Comment:
according to the guidelines, this needs to be put under `board-pinctrl.dtsi`. Also, you can directly extend pin nodes, ie use `&p3_3_scb2_uart_cts`.

Reply:
done